pages/index.vue

```diff
@@ -532,9 +532,24 @@
           }
         }
       },
+      setRouteQuerieState () {
```

Comment:
Can you change `Querie` to `Query`(spelling)

Reply:
done at [19a57e8](https://github.com/liyasthomas/postwoman/pull/118/commits/19a57e8ecf1289119049ab10c73a9cf313e52d0a). Thank you for your carefull review @abraham! I guess that was thinking in portuguese.. hahaha